src/main/java/org/openjdk/shenandoah/ToolbarPanel.java

```diff
@@ -191,6 +232,7 @@ private void setEnablePlaybackButtons(boolean b) {
         playPauseButton.setEnabled(b);
```

Comment:
Should the slider similarly be enabled/disabled for playback and _live_ mode?

Reply:
In the live mode, you can still move the slider but it won't create any effect. Is it better to disable it? Or leaving it this way is fine?

Reply:
Probably better to disable it. Having it usable but with no effect would be confounding for users.

Reply:
This should be fixed in the latest commit.

Reply:
Thank you!